Makefile

```diff
@@ -55,6 +55,16 @@ deps: tools
 tools:
 	@go get github.com/golang/dep/cmd/dep
 
+lint: prototool-bin
+	prototool lint
+
+prototool-bin:
+ifndef $(shell command -v prototool help > /dev/null)
+	@@curl -sSL https://github.com/uber/prototool/releases/download/v1.3.0/prototool-$(shell uname -s)-$(shell uname -m) -o $(HOME)/.bin/prototool
```

Comment:
Is there a better place than `$HOME/.bin` to store it? I wonder how this should work on CI.

Reply:
It is a Go project, you could also store it via go install in the GOPATH